### PR TITLE
Design Revamp Patch Up

### LINKED ISSUE #2131: v2.0.0-pre6: Scrolling bars appear white
- Scroll bars appear white 
- Should we do blue drawers with black background? 
- Tested on MacBook M1 Pro, macos Ventura 13.0.1

https://user-images.githubusercontent.com/47259243/216158873-2765ea23-2913-4680-8461-6954bf81c129.mov

### LINKED ISSUE #2133: v2.0.0-pre6: Swan Plugin Screen - all text underlined
When clicking on the Swan Plugin Screen, all the text is underlined in the screen.

(Tested on MacBook M1 Pro, macos Ventura 13.0.1, 10GB pruned node same machine)

https://user-images.githubusercontent.com/47259243/216163076-8a5c5906-73d8-409b-9254-858d41474e70.mov

### LINKED ISSUE #2137: v2.0.0-pre6: - Status “Receive (Unconfirmed)” & "Frozen"
When scrolling over the icon  “Receive (Unconfirmed)” - text doesnt fit in overlay label.
And the "Frozen" snowflake plus text doesnt fit space. Maybe move text into an overlay label.

Screenshot 2023-02-01 at 4 32 44 PM

### LINKED ISSUE #2152: v2.0.0-pre6: Import Wallet screen looks off.
- Import Wallet screen looks off, not yet styled.
- This was with a Spectrum connection
- Probably easy to fix, with prio/now

(Tested on MacBook M1 Pro, macos Ventura 13.0.1)

image

### LINKED ISSUE #2153: v2.0.0-pre6: Upload Keys screen looks off to the right.
- Upload Keys screen looks off to the right. 
- Probably should be aligned to left. 
- Should be Prio/Now, as important for clean onboarding flow.
- (Enter Pin screen could be improved, but that for later.)

https://user-images.githubusercontent.com/47259243/216435098-eda6ea52-92ce-4d9b-ad16-2b52b5c67bd7.mov

### PR DESCRIPTION
**Pull request by otkstdio**
Fixes #2153
Fixes #2137
Fixes #2131
Fixes #2133
Fixes #2152

### CODE REVIEW

File: src/cryptoadvance/specter/templates/includes/tx-table.html [hunk 1]
```diff
@@ -1018,8 +1018,8 @@ <h1>{{ _("Export transactions to CSV") }}</h1>
                         let frozenImgClone = txRow.frozenImg.cloneNode()
                         txRow.callbackaction.append(frozenImgClone)
                         // Adding "Frozen" in coin selection and UTXO overview
-                        txRow.callbackaction.append("Frozen")
-                        txRow.categoryCell.append("Frozen")
+                        // txRow.callbackaction.append("Frozen")
```
Comment (moneymanolis) [inline src/cryptoadvance/specter/templates/includes/tx-table.html:1021]: If we remove it, do we need some hover over / tooltip to explain the icon?

File: src/cryptoadvance/specter/static/style.css [hunk 2]
```diff
@@ -189,6 +194,24 @@
...
     animation: spin 1.5s linear infinite;
   }
+
+  /*
+   *
+   * ScrollBars
+   *
+   */
+
+  *::-webkit-scrollbar-track {
```
Comment (moneymanolis) [inline src/cryptoadvance/specter/static/style.css:204]: Still looks like this on Chrome for me:
grafik

and

grafik

File: src/cryptoadvance/specter/templates/includes/tx-table.html (hunk 1 above)
Comment (otkstdio) [reply src/cryptoadvance/specter/templates/includes/tx-table.html:1021]: No, i think it's clear enough with the icon

File: src/cryptoadvance/specter/static/style.css (hunk 2 above)
Comment (moneymanolis) [reply src/cryptoadvance/specter/static/style.css:204]: Ah wait, I am on your old branch. Wait a sec.

Comment (otkstdio) [reply src/cryptoadvance/specter/static/style.css:204]: This is very odd, i tested it on mine and it works with the new styles, on both Chrome & Arc.

Comment (moneymanolis) [reply src/cryptoadvance/specter/static/style.css:204]: I am on: Version 109.0.5414.119 (Official Build) (arm64)

**Comment by moneymanolis** [comment]
Bike-shed 1:
grafik

File: src/cryptoadvance/specter/templates/includes/tx-table.html (hunk 1 above)
Comment (moneymanolis) [reply src/cryptoadvance/specter/templates/includes/tx-table.html:1021]: OK

**Review by moneymanolis — APPROVED** [review]

File: src/cryptoadvance/specter/templates/device/new_device/new_device_keys.jinja [hunk 3]
```diff
@@ -160,7 +160,7 @@
                 <td><input id="xpubs-table-row-${xpubsTableRows}-purpose" name="xpubs-table-row-${xpubsTableRows}-purpose" type="text" value="${purpose}" placeholder="{{ _("XPUB purpose") }}" data-style="width: 90%; min-width: 100px;"></td>
                 <td id="xpubs-table-row-${xpubsTableRows}-derivation" data-style="min-width: 0px;">${derivation}</td>
                 <input id="xpubs-table-row-${xpubsTableRows}-derivation-hidden" name="xpubs-table-row-${xpubsTableRows}-derivation-hidden" type="hidden" value="${derivation}">
-                <td id="xpubs-table-row-${xpubsTableRows}-xpub" class="xpub scroll" data-style="overflow: auto;">${xpub}</td>
+                <td id="xpubs-table-row-${xpubsTableRows}-xpub" class="xpub scroll" data-style="overflow: auto;"><div class="table-key">${xpub}</div></td>
```
Comment (k9ert) [inline src/cryptoadvance/specter/templates/device/new_device/new_device_keys.jinja:163]: This is causing #2180 
Why using that div? Can we revert that for now?

**Pull request merged**